tardis/plasma/tests/test_nlte_excitation.py

```diff
@@ -73,83 +69,36 @@ def test_prepare_bound_bound_rate_matrix(
         r_lu_matrix,
         simple_beta_sobolev,
     )
-    desired_rate_matrix = [
-        [
-            [-4.41229578e10],
-            [1.09803977e10],
-            [8.87031593e08],
-            [1.83520728e08],
-            [5.71742068e07],
-        ],
-        [
-            [3.54409576e10],
-            [-3.64473689e11],
-            [1.32571818e11],
-            [1.04228424e10],
-            [2.27047121e09],
-        ],
-        [
-            [5.71505717e09],
-            [2.97836962e11],
-            [-1.41199954e12],
-            [6.39719360e11],
-            [5.08905487e10],
-        ],
-        [
-            [2.00818482e09],
-            [4.15382182e10],
-            [1.13720607e12],
-            [-3.74593514e12],
-            [1.98120242e12],
-        ],
-        [
-            [9.58758249e08],
-            [1.41181112e10],
-            [1.41334621e11],
-            [3.09560941e12],
-            [-2.03442061e12],
-        ],
-    ]
     # if this test fails the first thing to check is if the reshape in the
     # methods made a view or a copy. If it's a copy rewrite the function.
-    assert_allclose(
-        desired_rate_matrix,
-        np.array(actual_rate_matrix),
-        rtol=1e-6,
-    )
+    # TODO: allow rtol=1e-6
```

Comment:
I don't know if that specific tolerance is needed any more because the snapshot stores more significant figures than the parametrize above, right?

Reply:
In this case, it's saving it with the same precision as the parametrize here, so I think the tolerance is not as much needed I guess.